.gitignore

```diff
@@ -5,4 +5,3 @@ index.d.cts
 node_modules
 dist
 .yarn
-.env
```

Comment:
langchain build scripts remove this - need to figure out how to tell it to add this and `.eslintcache` to the `.gitignore`